content/CORE/UIReference/TopMenu/UIPreferences.md

```diff
@@ -12,80 +16,82 @@ aliases: /core/administration/uipreferences/
+| **Background 1** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **bg1** option in the **GENERAL** tab. |
+| **Background 2** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **bg2** option in the **GENERAL** tab. |
+| **Foreground 1** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **fg1** option in the **GENERAL** tab. |
+| **Foreground 2** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **fg2** option in the **GENERAL** tab. |
+| **Alt Background 1** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **alt-bg1** option in the **GENERAL** tab. |
+| **Alt Background 2** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **alt-bg2** option in the **GENERAL** tab. |
+| **Alt Foreground 1** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **alt-fg1** option in the **GENERAL** tab. |
+| **Alt Foreground 2** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **alt-fg2** option in the **GENERAL** tab. |
+| **Yellow** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **yellow** option in the **GENERAL** tab. |
+| **Orange** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **orange** option in the **GENERAL** tab. |
+| **Red** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **red** option in the **GENERAL** tab. |
+| **Magenta** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **magenta** option in the **GENERAL** tab. |
+| **Violet** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **violet** option in the **GENERAL** tab. |
+| **Blue** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **blue** option in the **GENERAL** tab. |
+| **Cyan** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **cyan** option in the **GENERAL** tab. |
+| **Green** | text input | Click the swatch to pick a color or enter a color hex. This value is used in for the **green** option in the **GENERAL** tab. |
+| **SUBMIT** | button | Save the current selections and create the new theme. |
+| **CANCEL** | button | Return to the **Preferences** screen without creating a new theme. |
 
 #### PREVIEW
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| Global Preview | toggle | Controls the visibility of the **Preview** widget. |
+| **Global Preview** | toggle | Controls the visibility of the **Preview** widget. |
 
```

Comment:
Same comment as above for removing the Style column. For a toggle, use something like, "Click the toggle to turn the display of the **Preview** widget on or off." or some variation on the behavior of the toggle.